lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -118,10 +86,7 @@ def define
       country_select :which_country?, additional_countries: additional_countries, exclude_countries: exclude_countries do
         on_response do |response|
           calculator.country = response
-        end
-
-        calculate :country_name do
-          (WorldLocation.all + additional_countries).find { |c| c.slug == calculator.country }.name
+          self.country_name = (WorldLocation.all + additional_countries).find { |c| c.slug == calculator.country }.name
```

Comment:
Could this be method on the calculator?

Reply:
Thank you - I've moved them to the calculator